### DISCUSSION TITLE
Matemática do Bob Esponja

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#352] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/matematica-do-bob-esponja/

**Comment by one-eyed-willy** [comment]
Eu nem tento entender a lógica do Bob Esponja.



É tudo debaixo d'água e ainda assim o Lula Molusco consegue tocar clarinete, o Bob Esponja/Patrick conseguem acender fogueiras, o Gary precisa tomar banho, as engenhocas do Plankton não dão curto-circuito...

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/matematica-do-bob-esponja/#comment_1114000647>):

> Eu nem tento entender a lógica do Bob Esponja.É tudo debaixo d'água e ainda assim o Lula Molusco consegue tocar clarinete, o Bob Esponja/Patrick conseguem acender fogueiras, o Gary precisa tomar banho, as engenhocas do Plankton não dão curto-circuito...

![null](<https://i.postimg.cc/tgfySvSF/matematica-do-bob-esponja-versao-alternativa.png>)

**Comment by najarin-the-loremaster** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/matematica-do-bob-esponja/#comment_1114011898>):

> [![null](<https://camo.githubusercontent.com/b3143c2e9d9301d2b816a300e35d4f9b8262d918e2ea84cf72ff79247795c1bb/68747470733a2f2f692e706f7374696d672e63632f74676679537653462f6d6174656d61746963612d646f2d626f622d6573706f6e6a612d76657273616f2d616c7465726e61746976612e706e67>)](<https://camo.githubusercontent.com/b3143c2e9d9301d2b816a300e35d4f9b8262d918e2ea84cf72ff79247795c1bb/68747470733a2f2f692e706f7374696d672e63632f74676679537653462f6d6174656d61746963612d646f2d626f622d6573706f6e6a612d76657273616f2d616c7465726e61746976612e706e67>)

Existe um livro chamado *SpongeBob SquarePants Trivia Book* que diz que Sra. Sirigueijo, mãe da Pérola é, de fato, uma baleia. Os pais da Pérola são siri e uma baleia porque o criador da série, Stephen Hillenburg, queria que cada personagem do elenco principal fosse um animal diferente.
